### PR TITLE
Add consul namespace documentation

### LINKED ISSUE #28: Update documentation about consul namespaces feature
https://github.com/fredrikhgrelland/vagrant-hashistack/issues/346

### PR DESCRIPTION
**Pull request by zhenik**
Documentation for https://github.com/fredrikhgrelland/vagrant-hashistack/pull/349

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -229,6 +230,27 @@ When ACLs are enabled in Nomad the bootstrap token will be available in vault un
 |     x     | consul_acl_default_policy        |  allow  |
 |           | consul_acl_default_policy        |  deny   |
 
+##### Enterprise - Namespaces
+
+[Consul namespaces](https://www.consul.io/docs/enterprise/namespaces) feature is available in enterprise version only.
+The next switches will enable [consul_namespaces_test.yml](https://github.com/fredrikhgrelland/vagrant-hashistack/blob/master/ansible/tests/enterprise/consul_namespaces_test.yml)
```
Comment (pdmthorsrud) [inline README.md]:
```suggestion
The switches below will enable [consul_namespaces_test.yml](https://github.com/fredrikhgrelland/vagrant-hashistack/blob/master/ansible/tests/enterprise/consul_namespaces_test.yml)
```

File: README.md [hunk 2]
```diff
@@ -229,6 +230,27 @@ When ACLs are enabled in Nomad the bootstrap token will be available in vault un
...
+
+```text
+consul_enterprise=true
+consul_acl=true
+consul_acl_default_policy=deny
+```
+
+Consul will come up with two additional namespaces ["team1", "team2"] and *admin token for these namespaces.
+
+`NB` Admin tokens have scope=global and policy=namespace-management.
```
Comment (pdmthorsrud) [inline README.md]:
```suggestion
> :warning: Admin tokens have scope=global and policy=namespace-management.
```

Comment (pdmthorsrud) [inline README.md]: When you say "policy=namespace-management", that is an already defined type of policy that consul comes with, right?  I couldn't find any documentation, but if you know of any, could you link to what the policy looks like?

Also, is it correct to say the scope is global, isn't it restricted to global within the namespace?

Comment (zhenik) [reply README.md]: fixed in b7ad85b

Comment (pdmthorsrud) [reply README.md]: Sweet!

**Review by pdmthorsrud — APPROVED** [review]
Rebase and then I'm good :D

**Pull request merged**